app/controllers/items_controller.rb

```diff
@@ -16,4 +17,25 @@ def create
       flash[:error] = "There was an error saving the item.  Please try again."
     end
   end
+
+  def destroy
+    @user = current_user
+    @item = Item.find(params[:id])
+    @item.destroy
+    flash[:notice] = "Item was removed from your list - Congratulations!"
+    redirect_to user_path(@user)
+  end
+
+  private
+
+  def item_params
+    params.require(:item).permit(:description)
+  end
+
+  def only_permit_current_user
+    unless user_signed_in? && (@user == current_user)
```

Comment:
in order to make this work, you need to first figure out who `@user` is (e.g. `@user = User.find(params[:user_id])`).  Another option would be to ignore the params[:user_id] entirely when creating new items and always set the item's user as the current user.